libs/base/src/lib/updaters/questsUpdater.ts

```diff
@@ -0,0 +1,47 @@
+import { useEffect } from 'react'
+import { getUnixTime } from 'date-fns'
+
+import { useUpdateQuestsMutation } from '@apps/artifacts/graphql/questbook'
+import { useAccount } from '@apps/base/context/account'
+import { useApolloClients } from '@apps/base/context/apollo'
+import { useAddQuestNotification } from '@apps/base/context/notifications'
+
+// TODO interval to update, or update with an exported hook?
```

Comment:
Currently it just updates when the user changes (including on connect), which is mostly ok